src/eba_handlers.eliom

```diff
@@ -112,10 +118,10 @@ let sign_up_handler' () email =
 let forgot_password_handler service () email =
   try_lwt
     lwt userid = Eba_user.userid_of_email email in
-    let text = "Hi,\r\nTo set a new password, please click on this link: " in
-    let act_key = generate_act_key ~service ~text email in
-    Eliom_reference.Volatile.set Eba_msg.activation_key_created true;
-    Eba_user.add_activationkey ~act_key userid
```

Comment:
I'll run it again just to be sure, but from the tests I made I had the perception that these modifications fixed the bug.
keep in touch :)